src/Sticky/index.jsx

```diff
@@ -0,0 +1,87 @@
+import React, { useEffect, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+
+const POSITION_VARIANTS = [
+  'top',
+  'bottom',
+];
+
+const Sticky = ({
+  position,
+  children,
+  offset,
+  className,
+  style,
+}) => {
+  const [isSticky, setIsSticky] = useState(false);
+  const ref = useRef();
```

Comment:
Curious if this component should also be using `React.forwardRef` in order for a consumer to pass an existing `ref` to `Sticky` like we do in other components, and otherwise fallback to its own `ref`?

Reply:
Yes, right, added `React.forwardRef`